insecure/attacknetwork/attackNetwork.go

```diff
@@ -54,31 +54,7 @@ func NewAttackNetwork(
 	// setting lifecycle management module.
 	cm := component.NewComponentManagerBuilder().
 		AddWorker(func(ctx irrecoverable.SignalerContext, ready component.ReadyFunc) {
-			// starts up gRPC server of attack network at given address.
```

Comment:
Just moving some codes from the constructor of `AttackNetwork` to its `start` method in order to satisfy some causality between the component's startup.